### DISCUSSION TITLE
Daily 06/12/2023

### DISCUSSION THREAD

**Discussion by jovanHillion** [Sinan-Karakaya/R-Type#45] [Daily]
Ajd je suis sur le test du scripting, réunion a 11h de 1h je reste jusqu'a 20h30

**Comment by mlbonniec** [comment]
#### Hier
* J'ai enfin réussi à build 🥳

#### Aujourd'hui
* Je commence à mettre en place l'architecture du client
* Je commence à mettre en place le réseau du client

**Comment by Sinan-Karakaya** [comment]
J'ai enfin pu commencer à gérer des entities depuis l'editor :)

Aujourd'hui je continue à faire encore plus d'intégrations, commencer le serializer, et l'assetManager
